src/features/rewards/box/index.tsx

```diff
@@ -113,7 +111,7 @@ export default class Box extends React.PureComponent<Props, {}> {
               </StyledRight>
               <StyledContent>
                 {
-                  isDisabled
+                  disabledContent
```

Comment:
we need to move control when to show disabled content to the parent compnent and just display content for disabled if provided